app/sections/posts/posts-list.component.ts

```diff
@@ -0,0 +1,33 @@
+import {Component} from 'angular2/core';
+import {DatePipe} from 'angular2/common';
+import {ContentfulService} from 'ng2-contentful/dist/src/services/contentful.service';
+
+@Component({
+  selector: 'posts',
+  template: <string> require('./posts-list.html'),
+  styles: [<string> require('./posts-list.css')],
+  providers: [ContentfulService],
+  pipes: [DatePipe]
+})
+export class PostsList {
+  private posts:any = [
+    {
+      title: 'Ignorance Survey in South Africa 2015',
+      date: new Date(),
```

Comment:
we have to add date ( or maybe more universal createdAt field ) to the contentful nodePage content type